tests/test_event_builder.py

```diff
@@ -322,7 +322,7 @@ def test_create_conversion_event__with_event_tags(self):
                                 event_builder.EventBuilder.HTTP_VERB,
                                 event_builder.EventBuilder.HTTP_HEADERS)
 
-  def test_create_conversion_event__with_invalid_event_tags(self):
+  def test_create_conversion_event__with_valid_string_revenue_event_tags(self):
```

Comment:
This test case was written to check invalid scenario. Please don't change the intent. 
Instead give an invalid revenue value instead of '4200'